.github/workflows/publish-on-release.yml

```diff
@@ -23,3 +23,28 @@ jobs:
       - run: npm publish --access public
         env:
           NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
+  build-docs:
```

Comment:
I would add `needs: publish-npm` to make sure we don't update the docs if npm publish fails for any reason.

Reply:
Good thinking, I've updated that.